lessons/lesson-2/tasks/task-8.js

```diff
@@ -10,15 +10,24 @@
 const human = Object.create({}, {
 	fullName: {
 		get() {
-			return `${this.firstName} ${this.lastName}`
+			if (!this.firstName) {
+				return this.lastName
+			} else if (!this.lastName) {
+				return this.firstName
+			} else {
+				return `${this.firstName} ${this.lastName}`
+			}
```

Comment:
Полагаю, что так будет удобнее, в случае если потребуется также вывести еще и отчество:

```suggestion
			return [this.firstName, this.lastName].filter(Boolean).join(' ')
```

Reply:
@PonomareVlad Исправил.